src/query/query-builder.ts

```diff
@@ -0,0 +1,60 @@
+import { ModelFilter, TransactionFilter } from "../filters";
+import { Metadata, QueryParams } from "../types";
+
+export interface BuildPathOptions {
+  filter: ModelFilter | TransactionFilter;
+  metadata: Metadata;
+  queryParams: QueryParams;
```

Comment:
Why the type and field is not called simply `Page`?

Reply:
Renamed